CONTRIBUTING.md

```diff
@@ -34,15 +34,11 @@ If you don't have npm, it comes with node. To get it via homebrew:
 `brew install node`
 
 To add or update a dependency:
-- `go get -u` to update the dependencies or `go get {package}` to add a dependency
+- `cat GLOCKFILE | grep -oE '[^[:space:]]+\.[^[:space:]]+' | sort | uniq | xargs go get -u` to update the dependencies or `go get {package}` to add a dependency
```

Comment:
`glock update` would be nice, agreed. Sadly `go get -u ./...` isn't quite enough because it won't update any of the `cmd`s we use